README.md

```diff
@@ -249,76 +262,102 @@ Include = /etc/pacman.d/cachyos-v4-mirrorlist
 Include = /etc/pacman.d/cachyos-mirrorlist
 ```
 
-Finally, update your system with CachyOS packages:
-
+**For AMD Zen4 CPUs:**
+```ini
+# CachyOS repositories (Zen4 optimized)
+[cachyos-znver4]
+Include = /etc/pacman.d/cachyos-znver4-mirrorlist
+[cachyos]
+Include = /etc/pacman.d/cachyos-mirrorlist
 ```
+
+**Finalize Installation:**
+
+```bash
+# Update package databases and system
 sudo pacman -Syu
 ```
-Enjoy improved system speed with CachyOS packages!
 
-## Debug packages
+🎉 **Congratulations!** Your system is now optimized with CachyOS packages.
 
-This is handled via debuginfod. Just add following file to
-`/etc/debuginfod/cachyos.urls`
+### Debug Packages
 
-```
-https://debuginfod.cachyos.org
+Debug symbols are available through debuginfod. Configure it by creating:
```

Comment:
Do we actually have debug packages for kernel at this point? I think until https://github.com/CachyOS/linux-cachyos/pull/268 will be landed there's no need to mention it.